src/input_utils.py

```diff
@@ -55,13 +57,13 @@ class PretrainInputConfig(object):
   # The fraction of tokens to mask for masked patch prediction (mpp) loss.
   mpp_fraction_to_mask = attr.ib(default=0.50)
 
-  # Maximum number of masked text tokens per batch.
-  mlm_max_selections_per_batch = attr.ib(default=480)
+  # Maximum number of masked text tokens per sequence.
+  mlm_max_selections_per_sequence = attr.ib(default=47)
```

Comment:
Nit: how do you determine the 47 and 98 numbers?

Reply:
Originally, I thought the variable `max_selections_per_batch` in [RandomItemSelector](https://github.com/googleinterns/multimodal-long-transformer-2021/blob/roylu-input/src/input_utils.py#L374) is used to determined the maximum number of selected masked tokens per "batch". However, I found that it is per "sentence". Thus, the 98 is from 196 (patch tokens) * 0.5 (patch masking probability) and 48 is from 314 (text tokens) * 0.15 (text masking probability).

Reply:
I see. Please try to incorporate these into the comment, which help readers to understand these 'magic' numbers.

Reply:
will do!